### ISSUE TITLE
File that displays unit information

### ISSUE THREAD

**Issue by maartenbrinkerink** [OSeMOSYS/osemosys_global#72]
As a non OSeMOSYS user it feels strange to not have insights in what units are being represented per variable. I suggest to add a file to the model folder that defines all units used per variable. I'm not sure if OSeMOSYS supports use of different units per variable but in that case it would also be good to have a unit input csv/config file that is being read before the model is composed.

**Comment by maartenbrinkerink** [comment]
@trevorb1 @tniet The above (adding units as user inputs) might be a bit ambitious and not worth the effort, but is there a way to easily output all used variables and their used units? If so we could perhaps add a table to the Read the Docs. If not then feel free to close this issue.

**Comment by trevorb1** [comment]
@maartenbrinkerink I agree, having user configurable units may not be required immediately. But def agree that adding units to readthedocs next to the model structure would be good!

**Comment by maartenbrinkerink** [comment]
You think that's a quick fix, as in, is there a command or so to print that out with little effort? Adding it to either milestone works for me, just trying to get things organized a bit. Especially for the somewhat older issues.

**Comment by maartenbrinkerink** [comment]
[Units overview.xlsx](https://github.com/user-attachments/files/17662688/Units.overview.xlsx)

Starting with and expanding to the above file units should be added to the read the docs.
